pufferd.go

```diff
@@ -72,11 +74,41 @@ func main() {
 	flag.BoolVar(&version, "version", false, "Get the version")
 	flag.BoolVar(&license, "license", false, "View license")
 	flag.BoolVar(&migrate, "migrate", false, "Migrate Scales data to pufferd")
+	flag.BoolVar(&uninstall, "uninstall", false, "Uninstall pufferd")
 	flag.StringVar(&configPath, "config", "config.json", "Path to pufferd config.json")
 	flag.Parse()
 
 	versionString := fmt.Sprintf("pufferd %s (%s %s)", VERSION, BUILDDATE, GITHASH)
 
+	if _, err := os.Stat(configPath); os.IsNotExist(err) {
+                if _, err := os.Stat("/etc/pufferd/config.json"); err == nil {
+			logging.Info("I did not find the config file you udes, but I did find a config file in the \"/etc/pufferd/config.json\" path. Using it.")
```

Comment:
Typo on this line, probably just easier to say "no config passed, defaulting to /etc/pufferd/config.json